src/config.js

```diff
@@ -46,22 +52,24 @@ const conf = convict({
   db: {
     host: {
       doc: 'Hostname where PostgreSQL is running',
-      format: 'url',
```

Comment:
I mean that it doesn't make a lot of difference to validate whether this configuration property is `path || url`; the connection to the db will fail with a clear enough error message anyway (and it could fail even if the property properly is a path or a url)